### ISSUE TITLE
Export registerCPO function

### ISSUE THREAD

**Issue by TuSKan** [mlr-org/mlrCPO#51]
In the article _https://github.com/mlr-org/mlrCPO/blob/master/info/developers.md_  on  section **CPO listing (listCPO.R))** you talk about how to register a new CPO for developers, but the function _registerCPO_ is not exported.

**Comment by mb706** [comment]
Since the document is primarily meant for developers that want to contribute to `mlrCPO` (possibly by adding new builtin CPOs), not for users (or developers of external packages that create their own CPOs) I am not making a distinction there between exported and internal functions. So far `registerCPO` only serves to list and document *builtin* CPOs.

You might have a point that it could be useful to even register "foreign" CPOs in the list so that `listCPO` finds them. I will think about it. (In that case, it might even be best to have `makeCPO` etc. call `registerCPO` automatically.)

**Comment by TuSKan** [comment]
You are correct, that document is for contributing developers. Sorry for my misunderstanding.
Very good idea to have `makeCPO` calling `registerCPO` internally.
Let me know if I can help. Thank you very much for developing this very useful package.
